browser/profiles/brave_profile_manager.cc

```diff
@@ -132,6 +133,7 @@ void BraveProfileManager::DoFinalInitForServices(Profile* profile,
   ProfileManager::DoFinalInitForServices(profile, go_off_the_record);
   brave_ads::AdsServiceFactory::GetForProfile(profile);
   brave_rewards::RewardsServiceFactory::GetForProfile(profile);
+  NewTabPageBrandedViewCounter::GetForProfile(profile);
```

Comment:
you don't need this here because `ServiceIsCreatedWithBrowserContext` returns true

Reply:
I added that override and removed this - TIL!